packages/mui/src/SelectWidget/SelectWidget.tsx

```diff
@@ -43,7 +43,13 @@ export default function SelectWidget<
 }: WidgetProps<T, S, F>) {
   const { enumOptions, enumDisabled } = options;
 
+  multiple = typeof multiple === "undefined" ? false : !!multiple;
+
   const emptyValue = multiple ? [] : "";
+  const _isEmpty = () =>
+    typeof value === "undefined" ||
+    (multiple && value.length < 1) ||
+    (!multiple && value === emptyValue);
```

Comment:
Why make this a function? You could simply assign it to a local variable and pass that instead?